changelogs/fragments/ansible-test-coverage-py26.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - ansible-test - Fix ``ansible-test coverage`` reporting sub-commands (``report``, ``html``, ``xml``) on Python 2.6.
```

Comment:
How is this related to Python 2.6? Or do you mean 3.6?

Reply:
The new tests uncovered a bug running the coverage reporting commands on Python 2.6. Since that bug exists in released versions of ansible-test I added a changelog fragment for the fix which is included in this PR. I didn't add a changelog fragment for the coverage traceback since it hasn't gone into any release.

Reply:
Ah, ok, makes sense :)